src/generator.cs

```diff
@@ -1978,7 +1975,7 @@ string MakeSig (string send, bool stret, MethodInfo mi, bool aligned, EnumMode e
 		try {
 			sb.Append (ParameterGetMarshalType (new MarshalInfo (this, mi) { IsAligned = aligned, EnumMode = enum_mode } ));
 		} catch (BindingException ex) {
-			throw new BindingException (ex.Code, ex.Error, ex,  "{0} in method `{1}'", ex.Message, mi.Name);
+			throw new BindingException (1078, ex.Error, ex, ex.Message, mi.Name);
```

Comment:
Why do we go from ex.Code to 1078? Is ex.Code always the same, if it is not, we are going to get the wrong message.